content/docs/intro/cloud-providers/fastly/_index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Fastly
+meta_desc: This page provides an overview of the Fastly Provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-fastly
+    weight: 5
+
+aliases: ["/docs/reference/clouds/fastly/"]
```

Comment:
Remove.

@stack72, in general, any **_new_** page you add should not have any aliases. We generally only need to use an alias when moving an existing page to a different URL path within the site.